.editorconfig

```diff
@@ -12,27 +12,3 @@ indent_size = tab
 tab_width = 4
 charset = utf-8
 trim_trailing_whitespace = true
-
-# Matches multiple files with brace expansion notation
-# Set default charset
-[*.{js,py}]
-charset = utf-8
-
-# 4 space indentation
-[*.py]
-indent_style = space
-indent_size = 4
```

Comment:
We do have python files in the test folder.

Reply:
True, but it does not add anything. We already have for `[*]

```
indent_style = space
indent_size = tab
tab_width = 4
```

With `tab_with=4` this is the same as `indent_size=4`